src/Component/GridInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid;
+
+use Sylius\Component\Grid\Builder\GridBuilderInterface;
+use Sylius\Component\Grid\Definition\Grid;
+
+interface GridInterface
+{
+    public static function getName(): string;
+
+    public static function getResourceClass(): string;
```

Comment:
A resource class is related to the SyliusResourceBundle, right? I think this should be moved to the ResourceBundle. There are also some other resource related options which has been implemented in the SyliusResourceBundle https://github.com/Sylius/SyliusResourceBundle/tree/master/src/Bundle/Grid

Reply:
ResourceInterface is related to Sylius Resource component 
https://github.com/Sylius/SyliusResourceBundle/blob/master/src/Component/Model/ResourceInterface.php